src/AddIns/Uno.UI.MediaPlayer.WebAssembly/HtmlMediaPlayer.cs

```diff
@@ -345,6 +346,13 @@ private void OnHtmlSourceLoaded(object sender, EventArgs e)
 			Duration = NativeMethods.GetDuration(_activeElement.HtmlId);
 		}
 		OnSourceLoaded?.Invoke(this, EventArgs.Empty);
+
+		if (_activeElement != null)
+		{
+			IsPause = NativeMethods.GetPaused(_activeElement.HtmlId);
+		}
+		_isPlaying = !IsPause;
```

Comment:
Should be we playing only if we set autoplay?

Reply:
In this branch there is no control if autoplay was requested (only in the complex one). 
In this branc the control is performed by the Extension layer sending the play.
In this case, we're getting it directly from the video's html tag if it's paused. Thus, if it is paused, _isPlaying updates to true. If it is not paused when loaded, then it is playing.
One possible refactor is unified the IsPause and _isPlaying in one single control state.